packages/cashier/src/modules/deposit-crypto/deposit-crypto.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect } from 'react';
+import { Divider } from '@deriv/components';
```

Comment:
Nice 🚀

Reply:
>Should we have a separate PR to master for cashier changes?
Not really, QA would be testing the cashier UI changes as part of merging feature branch to master. I'm discussing with Nhan to update the cashier design to reflect these changes in the mean time.